### PR TITLE
DF/DA-598(Login screen)

### PR DESCRIPTION
**Pull request by dacianfl**

### CODE REVIEW

File: marty_flutter/lib/themes/strings.dart [hunk 1]
```diff
@@ -0,0 +1,11 @@
+class AppStrings {
```
Comment (DragosMoldovan) [inline marty_flutter/lib/themes/strings.dart]: @dacianfl I don't think we still need this if we have localization strings.

File: marty_flutter/lib/screens/login/splash_screen.dart [hunk 2]
```diff
@@ -13,16 +18,30 @@ class SplashScreen extends StatefulWidget {
   State<StatefulWidget> createState() => _SplashScreenState();
 }
 
-class _SplashScreenState extends State<SplashScreen>
+class _SplashScreenState extends SubscriptionState<SplashScreen>
     with TickerProviderStateMixin {
-  final Duration _animationDuration = const Duration(milliseconds: 1000);
+  static final double startLogoSize = 50;
```
Comment (vladms) [inline marty_flutter/lib/screens/login/splash_screen.dart]: @dacianfl  Remove unused variables

File: marty_flutter/lib/screens/login/splash_screen.dart [hunk 3]
```diff
@@ -49,55 +77,91 @@ class _SplashScreenState extends State<SplashScreen>
...
+            }
+          },
+        ),
+      );
+
+  void _displayLogin() {
+    _fadeController.forward();
+    _loginFadeController.forward();
+    _splashTransitionController.forward();
+//    setState(() {
```
Comment (vladms) [inline marty_flutter/lib/screens/login/splash_screen.dart]: @dacianfl  Remove comments

File: marty_flutter/lib/screens/login/splash_screen.dart [hunk 4]
```diff
@@ -13,16 +18,30 @@ class SplashScreen extends StatefulWidget {
...
   void initState() {
     super.initState();
+    _viewModel = SplashScreenViewModel(
+      Input(
+        PublishSubject(),
+      ),
+      SharedPrefRepo(),
+    );
+    subscribeOnLogin();
     SystemChrome.setSystemUIOverlayStyle(
```
Comment (vladms) [inline marty_flutter/lib/screens/login/splash_screen.dart]: @dacianfl  I don't think we need this

**Pull request merged**